src/views/viewer/SearchOverlayView/SearchResults/SearchResults.tsx

```diff
@@ -25,7 +25,13 @@ const SearchResults: React.FC<SearchResultsProps> = ({ query }) => {
       return { channels: [], videos: [] }
     }
     const results = data
-    const videos = results.flatMap((result) => (result.item.__typename === 'Video' ? [result.item] : []))
+    const videos = results.flatMap((result) =>
+      result.item.__typename === 'Video' &&
+      result.item.mediaAvailability === AssetAvailability.Accepted &&
+      result.item.thumbnailPhotoAvailability === AssetAvailability.Accepted
+        ? [result.item]
+        : []
+    )
```

Comment:
Why do we do client-side filtering instead of specifying an argument for the query?

Reply:
I see now that the `search` query we have in the schema doesn't have all the possible arguments. Check https://hydra.joystream.org/graphql (docs on the right edge of the screen)